pallets/automation-time/src/lib.rs

```diff
@@ -654,14 +649,35 @@ pub mod pallet {
 			}
 		}
 
+		pub fn test_missing_tasks_remove_events(
```

Comment:
should i actually just use this function for benchmarking? or should i refactor the existing code to use this?